pdns/recursordist/ext/yahttp/meson.build

```diff
@@ -0,0 +1,3 @@
+module_dist = ['LICENSE', 'README.md']
+
+subdir('yahttp')
```

Comment:
I'm not sure why this file is a copy instead of a symlink?

Reply:
A simple botch, will fix